staging/src/k8s.io/apiserver/pkg/registry/rest/resttest/resttest.go

```diff
@@ -918,10 +920,15 @@ func (t *Tester) testDeleteWithResourceVersion(obj runtime.Object, createFn Crea
 	foo := obj.DeepCopyObject()
 	t.setObjectMeta(foo, t.namer(1))
 	objectMeta := t.getObjectMetaOrFail(foo)
-	objectMeta.SetResourceVersion("RV0000")
```

Comment:
I change this because we can't create an object with resourceVersion.

https://github.com/kubernetes/kubernetes/blob/d090d17cb5b3638e283b73173bb6a2ac0c54d54f/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L143-L149

the `s.versioner.PrepareObjectForStorage(obj)` will clear the value of `object.metadata.resourceVersion` :

https://github.com/kubernetes/kubernetes/blob/d090d17cb5b3638e283b73173bb6a2ac0c54d54f/staging/src/k8s.io/apiserver/pkg/storage/etcd3/api_object_versioner.go#L63-L72